commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java

```diff
@@ -35,6 +38,24 @@
 
   @Spec CommandSpec spec;
 
+  @Command(name = "export", description = "Export slashing db to json file")
+  public void exportSlashingDb(@Option(names = "--to") File output) {
+    final Eth2Runner runner =
+        new Eth2Runner(
+            config,
+            true, // must enforce slashing to be "on" or else no db connection is created
```

Comment:
This seems a bit hacky particularly in re-using the runner as are also creating the http server etc. just to do an export. Would like to not start the http server if we are doing the export. 

Otherwise you are going have to be careful not to run this on this same machine as the an existing web3signer or make sure your existing web3signer has a different port as we are using default ports in this mode. Could be useful for monitoring the export though if that takes a long time.

Reply:
done.